TravelPro.html

```diff
@@ -0,0 +1,295 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TravelPro</title>
+    <link rel="stylesheet" href="TravelPro.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+    <div class="navbar">
```

Comment:
Your use of semantic elements like `<header>, <footer>, and <div>` for different sections is good. However, consider using more specific HTML5 elements where appropriate, such as `<nav> `for the navigation links, `<section> `for distinct sections like services, destinations, etc., and `<article>` for individual items like service blocks or destination descriptions